client/disputes/evidence/file-upload.tsx

```diff
@@ -71,24 +72,32 @@ export const FileUploadControl = ( {
 					{ __( 'Upload file', 'woocommerce-payments' ) }
 				</FormFileUpload>
 
-				{ hasError ? (
+				{ hasError && (
 					<span className="upload-message is-destructive">
 						{ error }
 					</span>
-				) : (
-					fileName && (
-						<img
-							src={
-								'/wp-json' +
-								NAMESPACE +
-								'/file/' +
-								fileName +
-								'?as_account=0'
-							}
-							style={ { maxWidth: 100 } }
-							alt={ fileName }
-						/>
-					)
+				) }
+
+				{ ! hasError && (
```

Comment:
The error and preview conditions have become a bit complex. Perhaps this part can be extracted as a sub-component?

Reply:
Or a local variable?

Reply:
I have moved the File Upload component to the `components` folder as it is not Disputes only feature anymore.